src/main/java/net/dv8tion/jda/webhook/WebhookClientBuilder.java

```diff
@@ -34,7 +35,7 @@
 public class WebhookClientBuilder
 {
     public static final OkHttpClient.Builder DEFAULT_HTTP_BUILDER = new OkHttpClient.Builder();
-    private static final Pattern WEBHOOK_PATTERN = Pattern.compile("webhooks\\/(\\d+)\\/([\\w-]+)");
+    private static final Pattern WEBHOOK_PATTERN = Pattern.compile("(?:https?://)?(?:\\w+.)?discordapp\\.com/api(?:/\\d+)?/webhooks/(\\d+)/([\\w-]+)(?:/(?:\\w+)?)?");
```

Comment:
Did you mean to add the version here? because that would be `/api/v{d}/` rather than `/api/{d}/`